web_app/db/crud.py

```diff
@@ -573,3 +573,31 @@ def set_notification_allowed(
                     "is_allowed_notification": True
                 }
                 return self.create_telegram_user(user_data)
+            
+    def allow_notification(self, telegram_id: int) -> TelegramUser:
+        """
+        Update is_allowed_notification field to True for a specific telegram user
+        
+        Args:
+            telegram_id: Telegram user ID
+            
+        Returns:
+            Updated TelegramUser instance
+            
+        Raises:
+            SQLAlchemyError: If there's an error during the database operation
+        """ 
+        with self.Session() as session:
+            try:
+                user = session.query(TelegramUser).filter_by(telegram_id=telegram_id).first()
+                if not user:
+                    raise ValueError(f"User with telegram_id {telegram_id} not found")
+                
+                user.is_allowed_notification = True
+                session.commit()
+                session.refresh(user)
+                return user
```

Comment:
We don't need to return here anything, so remove `return user` and `session.refresh(user)`